components/scream/tests/scream_zm/CMakeFiles/3.14.5/CMakeCCompiler.cmake

```diff
@@ -0,0 +1,75 @@
+set(CMAKE_C_COMPILER "/usr/tce/bin/cc")
```

Comment:
What is the motivation for changing this file? These changes seem unrelated to getting ZM working.

Reply:
Yes, this should be removed. Besides not being portable (someone might not have `cc` in that path), it is also not safe in a CMake project to change the compiler _after_ the compilers have been inspected by cmake (either at the `project` call or `enable_language` call).

Reply:
I now see that there's a whole CMakeFiles folder. This looks like something you find in a build tree. I'm assuming you build (or built at least once) the code in the src tree, and ended up with these leftovers accidentally commited?

Reply:
Yes, this probably comes from building and testing.  Similar to what you get when you run `test-all-scream` and forget to delete the `ctest` directory.

Reply:
Ah, btw, to avoid seeing unstaged changes like the ctest directory, you can modify the `.git/info/exclude` file in your repo, and add that directory to the list. That file works like `.gitignore`, but it is _not part of the repo_, so it does not get staged/committed. It kind of works as a "local" `.gitignore`.